core/org.openjdk.jmc.agent/src/test/java/org/openjdk/jmc/agent/test/TestSetTransforms.java

```diff
@@ -72,40 +88,84 @@
 
 	@Test
 	public void testSetTransforms() throws Exception {
-		// Invoke retransform
-		MBeanServer mbs = ManagementFactory.getPlatformMBeanServer();
-		ObjectName name = new ObjectName(AGENT_OBJECT_NAME);
-		Object[] parameters = {XML_DESCRIPTION};
-		String[] signature = {String.class.getName()};
-		Class<?>[] clazzes = (Class<?>[]) mbs.invoke(name, "setTransforms", parameters, signature);
-		assertNotNull(clazzes);
-		if (Agent.getLogger().isLoggable(Level.FINE)) {
-			for (Class<?> clazz : clazzes) {
-				// If we've asked for verbose information, we write the generated class
-				TraceClassVisitor visitor = new TraceClassVisitor(new PrintWriter(System.out));
-				new CheckClassAdapter(visitor);
-				new ClassReader(TestToolkit.getByteCode(clazz));
-			}
+		boolean excpetionThrown = false;
+		try {
+			InstrumentMe.printHelloWorldJFR6();
+		} catch (Exception e) {
+			e.printStackTrace(System.err);
+			excpetionThrown = true;
 		}
+		assertFalse(excpetionThrown);
+
+		injectFailingEvent();
+		doSetTransfroms(XML_DESCRIPTION);
```

Comment:
doSetTransfroms -> doSetTransforms

Reply:
Fixed in https://github.com/openjdk/jmc/pull/23/commits/846cddff4c5d733e7544d096755566bc783cd5bf